sensor_stack/sparton/package.xml

```diff
@@ -0,0 +1,38 @@
+<package>
+  <name>sparton</name>
+  <version>1.0.0</version>
+  <description>sparton</description>
+  <maintainer email="teamauv@teamauv.local">vds</maintainer>
```

Comment:
@athityakumar Please change vds' email address to `vishnu@auviitkgp.local`. For people who have participated in the past, we will use `<firstname?@auviitkgp.local`. Change in the other `package.xml` files as well.